api/app_analytics/tasks.py

```diff
@@ -91,20 +94,27 @@ def track_feature_evaluation_v2(
 
 
 @register_task_handler()
-def track_feature_evaluation(
+def track_feature_evaluations_by_environment(
```

Comment:
I assume that the reason for the name change mainly to get around the fact that we don't have versioning for our tasks, but as a side effect, we're choosing to be more explicit with the naming anyway?